requirements.txt

```diff
@@ -1,4 +1,3 @@
-lxml
```

Comment:
why was this removed? It would seem that lxml isn't a dependency then?

Reply:
ncclient will install lxml as dependancy

Reply:
actually we need lxml for ncclient installation but as ncclient itself will take care of this i think this is not needed and also after removing this also we have lxml installed properly for more clarification please check this https://github.com/ncclient/ncclient/blob/master/requirements.txt

Reply:
also please check this https://ac974acc6c5551280ed4-b3f9d1886e750b23ce2ba8d42dd6b9f6.ssl.cf1.rackcdn.com/117/e5c1bf02cf428fd36a4c683d670e8a275f32e08b/check/ansible-test-units-iosxr-python35/396ada6/job-output.txt

Reply:
@pabelanger shall I add gate?